static/css/custom.css

```diff
@@ -416,15 +416,15 @@ h6 a.headline-link {
 
 .splash-text {
     color: white;
-    text-shadow: 0 0 20px black, 1px 0 20px black, -1px 0 20px black, 0 1px 20px black, 0 -1px 20px black;
+    text-shadow: 0 0 5px black, 0 0 10px black;
 }
 
-.splash-text h1,
-.splash-text h2,
-.splash-text h3,
-.splash-text h4,
-.splash-text h5,
-.splash-text h6 {
+.splash .splash-text h1,
+.splash .splash-text h2,
+.splash .splash-text h3,
+.splash .splash-text h4,
+.splash .splash-text h5,
+.splash .splash-text h6 {
```

Comment:
do we need a selector for every heading if there's only a h2? Or did I miss something?

Reply:
IME it's just convenient to have style available when adding other .splash-texts in the future.